api/HTMLMetaElement.json

```diff
@@ -142,6 +142,53 @@
           }
         }
       },
+      "media": {
```

Comment:
Can you add https://html.spec.whatwg.org/multipage/semantics.html#dom-meta-media as a spec URL?

Also if they're missing:
https://html.spec.whatwg.org/multipage/semantics.html#dom-meta-name
https://html.spec.whatwg.org/multipage/semantics.html#dom-meta-content
https://html.spec.whatwg.org/multipage/semantics.html#dom-meta-httpequiv

Reply:
Opened #13482 for the other spec URLs!